src/raw.rs

```diff
@@ -691,6 +691,24 @@ impl<'a> RawIobuf<'a> {
     Ok(())
   }
 
+  /// Both the limits and the window are [lo, hi).
+  #[inline]
+  pub fn expand_limits_and_window(&mut self, limits: (u32, u32), window: (u32, u32)) -> Result<(), ()> {
```

Comment:
The difference between this and `set_limits_and_window` should be documented somewhere.